### ISSUE TITLE
Add FreeDNS domains

### ISSUE THREAD

**Issue by bt90** [publicsuffix/list#88]
The letsencrypt project is using the public suffix list while checking for accidental and intentional abuse.(see https://community.letsencrypt.org/t/rate-limits-for-lets-encrypt/6769)

The result is that this prevents all users of the FreeDNS project(http://freedns.afraid.org/) from getting a free certificate due to said limits.

The problem is that we're talking about a list of ~90k domains which is being constantly updated with some being marked as 'private'(see http://freedns.afraid.org/domain/registry/). The addition of the public ones is still valid as they are in fact publicly used suffixes which are open for registration(e.g. mooo.com with ~400k subdomains). 

Not adding them would prevent ~8 million sites(check 'records' at http://freedns.afraid.org/domain/registry/) from using letsencrypt.

The sheer number of domains and their fluctuation would require automation. I'm already in contact with one of their admins(dnsadmin@afraid.org) and they would provide an export link for this purpose.

**Comment by TeZet2** [comment]
I strongly support this request! Accidentially (!? ;-) ) I was able to get a LE certificate for my mailserver at home and switched over from self signed to "real" SSL. Works like a charme. Tried to do the same with my owncloud server only one day later and I ran into the LE rate limits :-(. I strongly hope that either the rate limits will fall or we will see FeeDNS domains added to the public suffix list. I must admit that I am already addicted to LE ... .
